### DISCUSSION TITLE
Congratulations and Thank You

### DISCUSSION THREAD

**Discussion by askusWT** [blitz-js/blitz#2124] [Kind Words]
I was just about to ink a new Next/Chakra app when I stumbled over you, so first off, my apologies for being so uninformed. I'm blown away with how much and how intuitively you've expanded the concepts from next.js into bliz.js.

Context - This is my first step away from Create React App comfort zone.

I have a folder full of bookmarks, architecture design, headless options, Graph vs Rest trade offs - all the usual stuff, and as I was looking at it over the last week the question I mostly asked myself is why do I need to figure my way through 20 CMS decisions which add up to diddly squat in terms of real consequence. 

I wanted to jump into Next.js for the simplicity, beauty and complexity reduction, but it's almost too un-opinionated, it still has a cacophony of noise around it. 

A cynic says, I'm speaking too soon, you're a beta, I'll be adding to the issues list before I know it. 
But I say, the architecture is similar enough for me to work through it, and the hours you've saved me procrastination over build approach - and messing around with different options are already uncountable.

And so for that, I thank you. Time to dive in.

**Comment by flybayer** [comment]
😊 Thanks for the kind words!!
